Content.Client/Administration/UI/Tabs/BabyJailTab/BabyJailStatusWindow.xaml

```diff
@@ -0,0 +1,6 @@
+﻿<controls:BabyJailStatusWindow
+    xmlns="https://spacestation14.io"
+    xmlns:controls="clr-namespace:Content.Client.Administration.UI.Tabs.BabyJailTab"
+    Title="{Loc admin-ui-baby-jail-window-title}">
+    <Label Name="MessageLabel" Access="Public" Text="{Loc admin-ui-baby-jail-is-enabled}" />
```

Comment:
would make this a rich text label with the message boldened and the font in 20 or something